app/guides/layout.tsx

```diff
@@ -0,0 +1,22 @@
+"use server";
+
+import { getGuidePages } from "@/lib/server/cms-data";
+import { getSections } from "@/lib/nav-menu-builder";
+import NavMenu from "@/components/NavMenu";
+
+export default async function GuidesLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const guidePages = await getGuidePages();
+
+  return (
+    <div className="flex flex-col items-center">
+      <div className="flex flex-col items-start lg:w-[min(100%,1024px)] pl-2 lg:pl-0">
```

Comment:
I think moving the `w-min(100%,1024px)` to the page wrapping element on L15 and having the wrapper here just be 100% makes more sense - then all children of the page can use the wrapper as a guideline for how wide they are (you'll need to add `mx-auto` to center the content)

Check the Builder layout for an example.

Reply:
Also if a wrapper is returning a single element - the styles could just be passed down to that element instead